addon/webextension/background.js

```diff
@@ -77,10 +77,20 @@ class PersistentPageModificationEffect {
         display: inline-block;
       }
 
+      .donotdelete-revert {
+        transform: ScaleY(1);
+        transition: 1s ease-in;
+      }
+
+      /* after revert, revert the tooltip */
+      .donotdelete-revert .donotdelete-tooltip {
+        transform: scaleY(1) translateY(-50%);
```

Comment:
Please update these styles as we discussed to avoid the tooltip glitch:

```css
.donotdelete-revert {
  transform: scaleY(1);
  transition: transform 1s ease-in;
}

/* after revert, revert the tooltip */
.donotdelete-revert .donotdelete-tooltip {
  transform: scaleY(1) translateY(-50%);
  transition: transform 1s ease-in;
}
```

Reply:
done